README.md

```diff
@@ -211,6 +211,7 @@ conda activate reasoners
 
 Clone the repository and install the package:
 
+llama 2 have some alias problem so you may need to clone the forked llama1 at private repo. please check the setup.py
```

Comment:
No need to show this in readme. Just put a comment in setup.py is fine.

Reply:
I think it may be important to warn our group members or users with older version to re-setup for llama2 integration.